src/Compilers/Core/Portable/DiagnosticAnalyzer/SymbolDeclaredCompilationEvent.cs

```diff
@@ -16,15 +17,23 @@ internal sealed class SymbolDeclaredCompilationEvent : CompilationEvent
     {
         private readonly Lazy<ImmutableArray<SyntaxReference>> _lazyCachedDeclaringReferences;
 
-        public SymbolDeclaredCompilationEvent(Compilation compilation, ISymbol symbol, SemanticModel? semanticModelWithCachedBoundNodes = null)
+        public SymbolDeclaredCompilationEvent(
+            Compilation compilation,
+            ISymbol symbol,
+            ISymbolInternal symbolInternal,
```

Comment:
>ISymbolInternal symbolInternal, [](http://example.com/codeflow?start=12&length=31)

It looks like we always can get ISymbol from ISymbolInternal. Consider removing ```ISymbol symbol``` parameter.

Reply:
ah, good call.  will do.

Reply:
done.